run/runner.go

```diff
@@ -22,6 +22,8 @@ type Runner struct {
 	RepoPathFilters []string
 	RunResults      chan<- Result
 	RunQueue        <-chan bool
+
```

Comment:
empty line?

Reply:
Separating unexported fields from exported ones. Not sure if it's good practice but I've seen it around and I kind of like it.